avocado_vt/plugins/vt_joblock.py

```diff
@@ -114,9 +114,9 @@ def _lock(self, job):
 
     def pre_tests(self, job):
         try:
-            if job.test_suite is not None:
+            if job.test_suite.tests is not None:
```

Comment:
Hi @anascko, thanks for catching this. I'm not sure here, because looks like we have to handle here on Avocado-vt both, the current model and the previous one, right?

So maybe look for `if hasattr(test_suite, 'tests') ...` not sure.

Also, see #2639 proposed by @clebergnu.

Reply:
@beraldoleal thanks)
I missed this fix #2639